docs/topics/auth/customizing.txt

```diff
@@ -505,11 +507,12 @@ password resets. You must then provide some key implementation details:
 
     .. attribute:: USERNAME_FIELD
 
-        A string describing the name of the field on the User model that is
-        used as the unique identifier. This will usually be a username of
-        some kind, but it can also be an email address, or any other unique
-        identifier. The field *must* be unique (i.e., have ``unique=True``
-        set in its definition).
+        A string describing the name of the field on the User model
+        that is used as the unique identifier. This will usually be a
+        username of some kind, but it can also be an email address, or
+        any other unique identifier. The field *must* be unique (i.e.,
+        have ``unique=True`` set in its definition), unless you use a
+        custom authentication backend that can support it.
```

Comment:
This sentence doesn't have the context, should maybe be "unless you use a custom authentication backend that can support non-unique usernames."